tests/src/test/java/alluxio/testutils/master/MasterTestUtils.java

```diff
@@ -64,6 +69,19 @@ public static MasterRegistry createStandbyFileSystemMasterFromJournal() throws E
     return createFileSystemMasterFromJournal(false, null);
   }
 
+  /**
+   * Creates a new leader {@link FileSystemMaster} from journal along with its dependencies, and
```

Comment:
can you update the javadoc to say that this creates a copy of the journal, and then starts the fsm on that copied journal.